tests/integration_tests/style/test_gitlint.py

```diff
@@ -11,6 +11,7 @@ def test_gitlint():
     os.environ['LC_ALL'] = 'C.UTF-8'
     os.environ['LANG'] = 'C.UTF-8'
     try:
+        # TODO: update this line once the master branch is renamed
```

Comment:
we might forget this. should we create an issue to track it?

Reply:
Will do!